smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java

```diff
@@ -120,4 +123,88 @@ public void processMessage(Message message) {
         mucAsSeenByOne.leave();
         mucAsSeenByTwo.leave();
     }
+
+    @SmackIntegrationTest
+    public void mucDestroyTest() throws TimeoutException, Exception {
+
+        EntityBareJid mucAddress = JidCreate.entityBareFrom(Localpart.from("smack-inttest-join-leave-" + randomString),
+                                                            mucService.getDomain());
+
+        MultiUserChat muc = mucManagerOne.getMultiUserChat(mucAddress);
+        muc.join(Resourcepart.from("nick-one"));
+
+        final SimpleResultSyncPoint mucDestroyed = new SimpleResultSyncPoint();
+
+        UserStatusListener userStatusListener = new UserStatusListener() {
```

Comment:
If you instantiate a `DefaultUserStatusListener` then you could omit all those empty methods.

Reply:
Replaced `UserStatusListener` with `DefaultUserStatusListener`.